src/tls/connection.c

```diff
@@ -734,7 +734,8 @@ connection_thread_loop (Connection *self)
 }
```

Comment:
"we can the presence of the property" in commit message -- this sentence no verb